src/test/scala/com/cognite/sdk/scala/v1/AssetsTest.scala

```diff
@@ -263,6 +263,27 @@ class AssetsTest extends SdkTestSpec with ReadBehaviours with WritableBehaviors
     client.labels.deleteByExternalIds(Seq(externalId1, externalId2, externalId3))
   }
 
+  it should "update metadata on assets with empty map" in {
```

Comment:
Can we add into this test an update with None and with SetNull to see the behavior better hehe?  😁

Reply:
`setNull` is not supported for maps but I can add a test for `None` :D